smoke-tests-otel-starter/spring-boot-common/src/main/resources/META-INF/native-image/reflect-config.json

```diff
@@ -544,5 +544,706 @@
       "typeReachable": "com.zaxxer.hikari.util.Sequence$Factory"
     },
     "name": "java.util.concurrent.atomic.LongAdder"
+  },
```

Comment:
do we need all of those - of is it to be on the safe side?

Reply:
A first Kafka client hint has been added in a merged PR. Another hint is needed as seen in https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/11727. This another hint may not be sufficient.

All these hints should be automatically retrieved from the GraalVM metadata repo during the test executions. They are not because of a Gradle bug as mentioned above.